packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts

```diff
@@ -199,7 +203,7 @@ export interface SSICredentialIssuedSuccessPageConfig extends PageConfig {
     rightPaneButtonHeight?: string
 }
 
-export interface SSICredentialsLandingPageConfig extends PageConfig {
+export interface SSICredentialsLandingPageConfig extends PageConfig { // TODO
```

Comment:
Remove the comment or add a reference to a ticket meant to handle the issue

Reply:
added ticket, we do not have a project for this so it is CWALL

Reply:
Please add a link to the ticket in the comment